_posts/2015-08-06-Rust-1.2.md

```diff
@@ -0,0 +1,272 @@
+  [now working](https://github.com/rust-lang/rust/pull/26018), and produces a
+  33% speedup when bootstrapping on a 4 core machine. Parallel codegen is
+  particularly useful for debug builds, since it prevents some optimizations;
+  but it can also be used with optimizations as an effective `-O1` flag. It can
+  be activated by passing `-C codegen-units=N` to `rustc`, where `N` is the
+  desired number of threads.
+
+Cargo's performance has also improved dramatically:
+
+- Builds that do not require any recompilation ("no-op builds") for large
+  projects are much faster: for Servo, build time went from 5 seconds to 0.5
+  seconds.
+
+- Cargo now supports shared target directories that cache dependencies across
+  multiple packages, which results in significant build-time reduction for
+  complex projects.
+
+The 1.2 release also
+[introduces support](https://github.com/rust-lang/rust/pull/25350) for the MSVC
+(Microsoft Visual C) toolchain, as opposed to GNU variants. The upshot is that
+Rust code is now directly linkable against code built using the native Windows
+toolchain. The compiler bootstraps on MSVC, we have preliminary nightlies, and
+we are testing all rust-lang crates against MSVC. Unwinding support is not yet
+available (the process aborts on panic), but work is underway to land it.
+
+On the language side, Rust 1.2 marks the completion of the
+[dynamically-sized type (DST)](http://smallcultfollowing.com/babysteps/blog/2014/01/05/dst-take-5/)
+work, allowing smart pointers like `Rc` to seamless apply to arrays and trait
+objects, so that `Rc<[T]>` is fully usable. This final enhancement applies to
+all smart pointers in the standard library. Support for external smart pointer
+types is available in nightlies, and will be stabilized soon.
+
+### What's in 1.3 beta
+
+One of the most exciting developments during the 1.3 cycle was the introduction
+of the [Rustonomicon](https://doc.rust-lang.org/nightly/nomicon/), a new book
+covering "The Dark Arts of Advanced and Unsafe Rust Programming". While it's
+still in its early days, this book already provides indispensable coverage of
+some of Rust's more subtle aspects.
+
+The 1.3 cycle also saw additional focus on performance, though most wins here
+are within the standard library:
+
+- The substring matcher now uses a
+  [more efficient algorithm](https://github.com/rust-lang/rust/pull/26327).
+- There were
+  [improvements to zero filling](https://github.com/rust-lang/rust/pull/26849)
+  that speed up `Vec::resize` and `Read::read_to_end`
+- The implementation of `Read::read_to_end` has been
+  [specialized for `stdin` and `File`](https://github.com/rust-lang/rust/pull/26950),
+  resulting in additional speedups.
+- The `PartialEq` implementation on slices is now
+  [much faster](https://github.com/rust-lang/rust/pull/26884).
+
+We have also made strides in our Windows support, landing
+[preliminary support for Windows XP](https://github.com/rust-lang/rust/pull/26601). While
```

Comment:
s/support for/support for targeting/.

We still do not and probably never will support running rustc on XP.